zenpy/lib/api.py

```diff
@@ -1067,6 +1068,22 @@ def comments(self, ticket, include_inline_images=False):
         """
         return self._query_zendesk(self.endpoint.comments, 'comment', id=ticket, include_inline_images=repr(include_inline_images).lower())
 
+    @extract_id(Ticket, TicketComment)
+    def comment_redact(self, ticket, comment, text) -> TicketComment:
```

Comment:
Python2.7 doesn't like the typing unfortunately (causing the test to fail)

Reply:
Used it to trigger my autocompletion. Have removed it ;)